src/lucky_record/query_builder.cr

```diff
@@ -71,6 +72,12 @@ class LuckyRecord::QueryBuilder
     self
   end
 
+  def distinct_on(column)
```

Comment:
Can you add a type restriction for string | symbol so that someone doesn’t accidentally pass nil to it?